erpnext/accounts/report/statement_of_account/statement_of_account.html

```diff
@@ -1,26 +1,111 @@
-<h2 class="text-center">{%= __("Statement of Account") %}</h2>
-<h4 class="text-center">
-	{% if (filters.party_name) { %}
+<style>
+	.print-format.landscape {
+		margin: 100px;
+	}
+
+	.print-format td,
+	.print-format th {
+		padding: 12px !important;
+	}
+
+	.customer-info {
+		color: #366092;
+		margin: 20px 0;
+		text-align: center;
+	}
+
+	.print-format .customer-info p {
+		color: #366092;
+		font-size: 16px;
+	}
+
+	.customer-info h2 {
+		font-weight: bold;
+		margin: 0px;
+		text-transform: uppercase;
+	}
+
+	.table-bordered thead tr th,
+	.table-bordered tbody tr td {
+		border: 1px solid #0000ff;
+	}
+
+	.table-bordered thead tr th {
+		background-color: #0000ff !important;
+		color: #ffffff;
+		font-size: 15px;
+		padding: 10px;
+		text-align: center;
+		text-transform: uppercase;
+	}
+
+	.table-bordered tbody tr:nth-child(odd) td {
+		background-color: #cfe2f3;
+	}
+
+	.address,
+	.statement-date {
+		display: inline-block;
+	}
+
+	.address {
+		width: 50%;
+	}
+
+	.statement-date {
+		float: right;
+		width: 50%;
+	}
+
+	.address p,
+	.statement-date .row label,
+	.statement-date .row .value,
+	.print-date {
+		color: #948a8a;
+		font-size: 15px;
+	}
+</style>
+
+<div class="row">
+	<div class="col-xs-6 address">
+		<p> As per company letter head address</p>
+	</div>
+	<div class="col-xs-6 statement-date ">
+		<div class="row">
+			<div class="col-xs-6 text-right">
+				<label>Statement Date : </label>
+			</div>
+			<div class="col-xs-6 value text-right">
+				{%= frappe.datetime.str_to_user(filters.from_date) %} {%= __("to") %}
+				{%= frappe.datetime.str_to_user(filters.to_date) %}
+			</div>
+		</div>
+	</div>
+</div>
+
+<div class="customer-info">
+	<h2>{%= __("Statement of Account") %}</h2>
+	<p>
+		{% if (filters.party_name) { %}
 		{%= filters.party_name %}
-	{% } else if (filters.party && filters.party.length) { %}
+		{% } else if (filters.party && filters.party.length) { %}
 		{%= filters.party %}
-	{% } else if (filters.account) { %}
+		{% } else if (filters.account) { %}
 		{%= filters.account %}
-	{% } %}
-</h4>
-
-<h6 class="text-center">
-	{% if (filters.tax_id) { %}
-	{%= __("Tax Id: ")%}	{%= filters.tax_id %}
-	{% } %}
-</h6>
-
-<h5 class="text-center">
-	{%= frappe.datetime.str_to_user(filters.from_date) %}
-	{%= __("to") %}
-	{%= frappe.datetime.str_to_user(filters.to_date) %}
-</h5>
-<hr>
+		{% } %}
+	</p>
+
+	<p>
+		{% if (filters.tax_id) { %}
+		{%= __("Tax Id: ")%} {%= filters.tax_id %}
+		{% } %}
+	</p>
+
+	<!-- <p>
+		{% frappe.db.get_value("Dynamic Link", {"link_doctype": "Customer", "link_name": filters.party[0]},"parent") %}
+	</p> -->
```

Comment:
We still need to figure out how to fetch the addresses, I'll have to look at this.

Reply:
@Alchez Can we not merge it till then?

Reply:
No point merging it, really, without the entire format done